docs/tutorials/first_app/blinky_compiler.md

```diff
@@ -1,70 +1,81 @@
-## Blinky on the Arm Mbed Online Compiler
+## Online Compiler
 
-This tutorial builds Blinky using the Arm Mbed Online Compiler, which allows you to build Mbed OS applications without installing a toolchain on your own machine.
+### Setup
 
-<span class="tips">Please create a <a href="https://os.mbed.com/account/signup/" target="_blank">developer account</a>. It's free, and we don't spam.</span>
+#### Create an Mbed developer account
+Go to os.mbed.com and [create an account](https://developer.mbed.org/account/signup/?next=%2F)
 
-### Importing Blinky
+#### Setup Environment
+- Plug your mbed board into your computer and open its USB device folder
+- Double click on the MBED.HTM file (this will add your mbed platform to the online compiler)
 
-To get Blinky into the Mbed Online Compiler, click the **`Import into Mbed IDE`** button below:
+If you do not have an Mbed board, go to [os.mbed.org/platforms](http://os.mbed.org/platforms), select a board and click the “Add to your mbed Compiler” button.
+<span class="images">![](https://sarahmarshy.github.io/img/add-to-compiler.png)
+</span>
 
-[![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky/)](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky/file/tip/main.cpp)
+### Code
 
-You're taken to the online IDE, and the **Import Program** dialog box opens:
+#### Import 
+Visit the mbed-os [blinky example repository](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-blinky/) and click the "Import into Compiler" button.
+ 
+<span class="images">![](https://sarahmarshy.github.io/img/import-compiler.png)
+</span>
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import_dialog.png)<span>Importing Blinky</span></span>
+#### Compile
+Click on the "Compile" button, your browser will then download the program as a executable file.
 
-The import mechanism offers a default name, but you're free to change it. When you're done, click **Import**.
+<span class="images">![](https://sarahmarshy.github.io/img/compile.png)
+</span>
 
-### Viewing Blinky
+#### Program
+Open the folder where the executable file was downloaded, then click and drag (or copy and paste) the file to your mbed board's USB device folder.
 
-The imported Blinky has two interesting parts:
+Once the file has been flashed to the board, press the board's "reset" button and you should now see the LED blinking.
 
-- `main.cpp`, where the Blinky-specific code is. You can double-click the file in the navigation pane on the left to view the code.
-- `mbed-os`, where the Arm Mbed OS codebase is.
 
-Later we'll compile the code; this will take both of these parts and create a single application file from them.
+### Debug
```

Comment:
Again, is debugging part of the scope of the quickstart?

Reply:
yes, this is very limited in its scope, thus i feel it is appropriate